packages/styles/forms.css

```diff
@@ -437,8 +437,11 @@ textarea.Field--has-error:focus:hover,
 }
 
 .Radio__overlay--disabled.Icon--radio-checked,
-.Checkbox__overlay--disabled.Icon--checkbox-checked {
+.Radio__overlay--disabled.Icon--radio-unchecked,
+.Checkbox__overlay--disabled.Icon--checkbox-checked,
+.Checkbox__overlay--disabled.Icon--checkbox-unchecked {
   color: var(--field-icon-checked-disabled-color);
+  cursor: default;
```

Comment:
Sorry, one last request: These are split into 2 different style rules, one for checked checkboxes and one for unchecked checkbox so the change should set `cursor: default` onto both instead of adding new selectors:

```css
.Radio__overlay--disabled.Icon--radio-checked,
.Checkbox__overlay--disabled.Icon--checkbox-checked {
  color: var(--field-icon-checked-disabled-color);
  cursor: default;
}

.Checkbox__overlay--disabled.Icon--checkbox-unchecked,
.Radio__overlay--disabled.Icon--radio-unchecked {
  color: var(--field-icon-unchecked-disabled-color);
  cursor: default;
}
```

Reply:
Good now? @scurker

Reply:
Almost, see review comment

Reply:
Oops. 😅